README.md

```diff
@@ -32,7 +32,9 @@ The following role variables are used for `node_exporter`'s service configuratio
 
 - `node_exporter_args`: A list of arguments to pass to the `node_exporter` service, one
   argument per list item.
+- `node_exporter_group`: Group for the to run `node_exporter_user`.
```

Comment:
"Group for the to run" is missing a word.

Reply:
Fixed in 90bbddb. Please forgive the force-push, I wanted to fix this and deal with the history rewriting at the same time.